src/__Private/LinterCLI.php

```diff
@@ -156,6 +156,19 @@ private function lintPath(
         return 1;
       }
     } else {
+      foreach ($roots as $root) {
+        $path = \realpath($root);
+        if (\is_dir($path)) {
+          $config_file = $path.'/hhast-lint.json';
+          if (\file_exists($config_file)) {
+            \fwrite(
+              \STDERR,
+              "Warning: PATH arguments contain a hhast-lint.json\n",
```

Comment:
Logic looks correct, but it would be good to make the message explain why this is a potential problem.

Reply:
and what they might want instead (`cd $dir; vendor/bin/hhast-lint`)